kpconv_torch/test.py

```diff
@@ -101,7 +101,10 @@ def main(args):
 
     # Initialize configuration class
     config = Config()
-    config.load(chosen_log)
+    config.set_infered_file_path(args.filename)
+    config.set_chosen_log_folder(args.chosen_log)
+    config.command = "test"
+    config.load()
```

Comment:
les paramètres de la CLI sont affectés depuis l'extérieur de la fonction `load` et `chosen_log` n'est donc plus passé en paramètre de la fonction `load()`.

Reply:
Basically, each log folder corresponds to an experiment, *i.e.* a training session. Each training session has its own configuration. Why does `chosen_log` have to be removed from the `load` function signature?

Reply:
Afterwards, I agree. I came back to the initial design, after understanding better the function of the Config class. Calls of S3DISDataset and S3DISSampler (and similarly for other datasets) have been standardized. A code factorization would be possible, delegating some of the parameters to the mother class. It will be done in a coming commit.